### DISCUSSION TITLE
Router visit, get and other always have 409

### DISCUSSION THREAD

**Discussion by rizkhal** [inertiajs/inertia#1691] [Help (Vue)]
Hi.

I'm trying to build an small packages with vue3 and inertia, but i always have 409 when trying to use router inside the packages.

the full source code is here https://github.com/rizkhal/inerui/issues/1

i think its because when inertiajs router trying to fetch from laravel, the router has conflict from my my package and inertia inside playground, but i don't know how to find the actual request between that.

Thanks 🍻

**Comment by mrvnklm** [comment]
Having the same issue, which always causes full page reload.

**Comment by Taro-oka** [comment]
I had the same issue but I changed the way to import Inertia 
from 
`import {Link} from '@inertiajs/vue3';`
to 
`import { Link } from '@inertiajs/inertia-vue3';` 
then it solved the issue.
